src/Microsoft.VisualStudio.Services.Agent/Capabilities/AgentCapabilitiesProvider.cs

```diff
@@ -117,7 +117,7 @@ private static string GetDarwinVersionString()
             }
             else
             {
-                return $"11.{version.Major - 20}";
+                return $"11.{version.Minor}";
```

Comment:
It doesn't look like this method is ever going to return 12.*, or am I missing something?

Reply:
Very good point, fixing that